master/internal/experiment.go

```diff
@@ -505,9 +530,15 @@ func (e *experiment) processOperations(
 			updatedTrials[op.RequestID] = true
 		case searcher.Shutdown:
 			if op.Failure {
-				e.updateState(ctx, model.StoppingErrorState)
+				e.updateState(ctx, model.StateWithReason{
+					State:               model.StoppingErrorState,
+					InformationalReason: "hp searcher failure",
+				})
 			} else {
-				e.updateState(ctx, model.StoppingCompletedState)
+				e.updateState(ctx, model.StateWithReason{
+					State:               model.StoppingCompletedState,
+					InformationalReason: "hp searcher shutdown",
```

Comment:
maybe here and elsewhere "hp search" rather than "hp searcher". and I would say "completed" not shutdown here